docs/TektonResult.md

```diff
@@ -1,58 +1,22 @@
-<!--
```

Comment:
Should this comment still be retained?
The effect after the modification is not very consistent with other pages.

![Shot 2025 02 19 at 21 15 40](https://github.com/user-attachments/assets/30d447b4-cd5e-4f7c-a06f-b24e013ef88d)

Reply:
yes, these changes are done by mistake and remove it